src/constants/index.ts

```diff
@@ -69,3 +71,8 @@ export const FUSD = new Token(
 export const BASES_TO_CHECK_TRADES_AGAINST = {
   [ChainId.FUSE]: [WFUSE, DAI, USDC, USDT, WBTC, WETH]
 }
+
+export const TOKEN_MAP = {
+  [ZERO_ADDRESS]: WFUSE_ADDRESSS,
+  '0x249be57637d8b013ad64785404b24aebae9b098b': '0x620fd5fa44be6af63715ef4e65ddfa0387ad13f5'
```

Comment:
Can you use constacts for this? like `USDC.address.toLowerCase()`
Didn't changed this myseelf cause wasn't sure how to handle this back then